src/index.html

```diff
@@ -22,6 +22,394 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a
+          href="#"
+          class="header__logo-link"
+        >
+          <img
+            src="images/logo.png"
+            alt="header__logo"
```

Comment:
The `alt` attribute should be descriptive of the image content. Instead of 'header__logo', consider something like 'Company Logo' to provide better context for screen readers and search engines.